db/migrate/20200705102418_add_sample_rate_and_bit_depth_to_audio_file.rb

```diff
@@ -0,0 +1,22 @@
+class AddSampleRateAndBitDepthToAudioFile < ActiveRecord::Migration[6.0]
+  def up
+    add_column :audio_files, :sample_rate, :integer
+    add_column :audio_files, :bit_depth, :integer
+
+    AudioFile.find_each do |af|
```

Comment:
This migration would take hours, if not days. That's not acceptable. If we want this (which I'm still not convinced about), a job will need to be scheduled for each audio_file, so that the app is usable while these values are calculated.

Reply:
It should take about 20 to 30 minutes if I calculated it right.
But even then your comment is still true, I've updated the PR